src/platform/windows_winusb/enumeration.rs

```diff
@@ -98,13 +98,27 @@ pub fn probe_device(devinst: DevInst) -> Option<DeviceInfo> {
 
     interfaces.sort_unstable_by_key(|i| i.interface_number);
 
+    let bus_devs = cfgmgr32::list_interfaces(GUID_DEVINTERFACE_USB_HOST_CONTROLLER, None)
+        .iter()
+        .flat_map(|i| get_device_interface_property::<WCString>(i, DEVPKEY_Device_InstanceId))
+        .flat_map(|d| DevInst::from_instance_id(&d))
+        .flat_map(|d| d.children())
+        .map(|d| d.instance_id().to_string())
+        .enumerate()
+        .map(|v| (v.1, (v.0 + 1) as u8))
+        .collect::<HashMap<String, u8>>();
```

Comment:
The existing bus number using `DEVPKEY_Device_BusNumber` is indeed wrong (https://github.com/kevinmehall/nusb/issues/70) but as mentioned in that issue I'm considering just exposing the bus ID as a string rather than trying to map it to a small integer.